docs/proposals/excess_reserve_capacity.md

```diff
@@ -1,55 +1,74 @@
 # Excess Reserve Capacity - WIP
```

Comment:
This is not WIP anymore, I forgot to remove it later. Can you please use this PR remove `WIP` tag.